xbmc/platform/win32/XTimeUtils.cpp

```diff
@@ -19,154 +19,6 @@ namespace KODI
 {
 namespace TIME
 {
-namespace
```

Comment:
Seems this file is almost "empty" now, then `xbmc/platform/win32/XTimeUtils.cpp` can be deleted?

And remove also from:
https://github.com/xbmc/xbmc/blob/777b96257d609454a4b4716a2b9106d5a73f7fcd/xbmc/platform/win32/CMakeLists.txt#L14

But then the "daylight saving" bug will probably come back:

> Remarks
> To account for daylight saving time when converting a file time to a local time, use the following sequence of functions in place of using FileTimeToLocalFileTime:
> 
> [FileTimeToSystemTime](https://learn.microsoft.com/en-us/windows/desktop/api/timezoneapi/nf-timezoneapi-filetimetosystemtime)
> [SystemTimeToTzSpecificLocalTime](https://learn.microsoft.com/en-us/windows/desktop/api/timezoneapi/nf-timezoneapi-systemtimetotzspecificlocaltime)
> [SystemTimeToFileTime](https://learn.microsoft.com/en-us/windows/desktop/api/timezoneapi/nf-timezoneapi-systemtimetofiletime)

https://learn.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-filetimetolocalfiletime#remarks

This bug affects at hashes calculation at scanning libraries as time of each file is considered for hash.

See: https://github.com/xbmc/xbmc/pull/23631

I will try to test this in the next few days but it is not easy.

Probably the fix is continue use "custom" `FileTimeToLocalFileTime` implementation that internally calls:

```
FileTimeToSystemTime()
SystemTimeToTzSpecificLocalTime()
SystemTimeToFileTime()
```

Maybe move to `CWIN32Util::` or call in place instead of `FileTimeToLocalFileTime`

Reply:
Thanks for the review! I will check it too, especially given the current implementation makes up 1 hour on Win11 but is OK on Server 2019 and 2022: https://forum.kodi.tv/showthread.php?tid=378981&pid=3211078#pid3211078

Reply:
Also it is worth mentioning that Windows and UWP builds now use C++20 native `std::chrono`, not Howard's date library.

Reply:
https://github.com/microsoft/STL/issues/4997

Reply:
Another viable solution is to port code from ICU wintz into Howard's date library and ditch (for now!) c++20 implementation.

Reply:
do you mean that Kodi 22 should have a minimum requirement starting from Windows 10 22H2?

if so Win10 1903 is EOL, also Windows 8.1 is dead, imo higher OS version requirement could be considered

Reply:
Windows 10 still has a huge user base, way more than windows 11. I think the rest are infinitely small by comparison now.